### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by schu**

### CODE REVIEW

**Comment by schu** [comment]
CircleCI is not configured yet, please ignore.

File: builder [hunk 1]
```diff
@@ -90,5 +93,8 @@ EOF
 sed -e "s/{{kernel_version}}/${kernel_version}/" "${dir}/manifest.tmpl.json" >"${aci_dir}/manifest"
 
 # build aci
-tar -czf "${target_aci}" -C "${aci_dir}" .
+tar -czf "${target_aci}" \
+  --exclude ".install" \
+  --exclude "..install.cmd" \
```
Comment (krnowak) [inline builder:98]: dot dot install.cmd? Is there a typo?

Comment (schu) [reply builder:98]: No, actually named `..install.cmd`. A Linux build system thing.

Comment (schu) [reply builder:98]: > A Linux build system thing.

That's why I don't `find .... -delete` the files from the tree.

**Pull request merged**